dash/dash.py

```diff
@@ -216,6 +216,9 @@ def add_url(name, view_func, methods=('GET',)):
             '{}<path:path>'.format(self.config['routes_pathname_prefix']),
             self.index)
 
+        add_url('{}_favicon'.format(self.config['routes_pathname_prefix']),
```

Comment:
Could this route be `{}_favicon.ico`? I feel like this would make it clearer that the route is serving a .ico file directly

Reply:
Yes probably clearer thanks.